raiden/network/transport/matrix.py

```diff
@@ -344,23 +342,25 @@ def start(
         self,
         raiden_service: RaidenService,
         message_handler: MessageHandler,
-        fetch_since_token: str,
+        prev_auth_data: str,
     ):
         if not self._stop_event.ready():
             raise RuntimeError(f'{self!r} already started')
         self._stop_event.clear()
         self._raiden_service = raiden_service
         self._message_handler = message_handler
 
-        # Initial login sync will be performed w/o sync token to get all meta events
-        self._login_or_register()
-        self.log = log.bind(current_user=self._user_id, node=pex(self._raiden_service.address))
-
         # Initialize the point from which the client will sync messages
-        if fetch_since_token:
-            prev_user_id, _, prev_sync_token = fetch_since_token.partition('/')
-            if prev_user_id == self._user_id:
-                self._client.set_sync_token(prev_sync_token)
+        if prev_auth_data and prev_auth_data.count('/') == 1:
+            prev_user_id, prev_access_token = prev_auth_data.split('/')
```

Comment:
Minor nitpick: You could use `prev_user_id, _, prev_access_token = prev_auth_data.partition('/')` instead. Would also prevent this from breaking in case the auth token should contain a slash at some time in the future.